cin_validator/rules/cin2022_23/rule_4008.py

```diff
@@ -85,13 +85,13 @@ def validate(
     # we can now map the suffixes columns to their corresponding source tables such that the failing ROW_IDs and ERROR_IDs exist per table.
     df_cpp_issues = (
         df_ci.merge(df_merged, left_on="ROW_ID", right_on="ROW_ID_ci")
-        .groupby("ERROR_ID")["ROW_ID"]
+        .groupby("ERROR_ID", group_keys=True)["ROW_ID"]
```

Comment:
Hi @willLPD2I these should be False instead.

Reply:
done